src/wcag-taxonomy-provider.test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Sarif from './sarif/sarif-2.1.2';
+import { getWcagTaxonomy } from './wcag-taxonomy-provider';
+
+describe('WCAGTaxonomyProvider', () => {
+    it('creates a Sarif.ToolComponent object with WCAG property information', () => {
+        const sortedTagsStub: string[] = [];
+        const expectedResultsStub: Sarif.ToolComponent = {
+            name: 'WCAG',
+            fullName: 'Web Content Accessibility Guidelines (WCAG) 2.1',
+            organization: 'W3C',
+            informationUri: 'https://www.w3.org/TR/WCAG21',
+            version: '2.1',
+            guid: '',
```

Comment:
We might as well go ahead and include a real guid here for completeness, imo. The one from the sample file (`guid: 'ca34e0e1-5faf-4f55-a989-cdae42a98f18'`) is fine to use here, I generated it afresh when I made the sample file.

Reply:
(we'll want to extract it into a reusable constant at some point, but it's okay to defer that until the later PR where you'll need to re-use it for the first time)